WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,8 +1,4 @@
-- In-app site domain purchases
-- Drag-and-drop blocks in the editor
-- Better color contrast on Cover blocks
-- Buttons block formatting options stay visible
-- Separate Quick Start tasks for new vs existing sites
-- Copy Link for published pages
-- Link To/Image Size settings in Gallery blocks
-- No tour banner in unsupported block editor
+* [*] Block Editor: A11y: Improve text read by screen readers for BottomSheetSelectControl [https://github.com/WordPress/gutenberg/pull/41036]
+* [*] Block Editor: Add 'Insert from URL' option to Image block [https://github.com/WordPress/gutenberg/pull/40334]
+* [*] Quick Start: The "Get to know the app" card and Quick start tour list has a fresh new look [https://github.com/wordpress-mobile/WordPress-Android/pull/16629,https://github.com/wordpress-mobile/WordPress-Android/pull/16639]
```

Comment:
cc @leandroalonso just a confidence check that all of those apply equally to WP and JP (even though I'm pretty sure they all do) 🙂

Reply:
You're right! They all do. :)